index.html

```diff
@@ -0,0 +1,166 @@
+            <ul>
+                <li><i class="bi bi-twitter"></i></li>
+                <li><i class="fa fa-facebook"></i></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+    </header>
+
+    <section>       
+        <nav>
+            <div class="navbar">
+                <div class="container nav-container">
+                    <input class="checkbox" type="checkbox" name="" id="" />
+                    <div class="hamburger-lines">
+                        <span class="line line1"></span>
+                        <span class="line line2"></span>
+                        <span class="line line3"></span>
+                        <div><img src="img/CS-logo.png" alt=""></div>
+                    </div>
+                    
+                    <a href="index.html"><img src="img/CS-logo.png" alt="Page Logo"></a>
+                    <ul class="nav-items">
+                        <li class="nav-item"><a href="about.html">About</a></li>
+                        <li class="nav-item"><a href="#">Program</a></li>
+                        <li class="nav-item"><a href="#">Join</a></li>
+                        <li class="nav-item"><a href="#">Sponsor</a></li>
+                        <li class="nav-item"><a href="#">News</a></li>
+                    </ul>
+                    <button class="nav-button">CS Campaign</button>
+                </div>
+            </div>
+        </nav>
+
+        <div class="header-content-container">
+            <div class="header-content">
+                <p class="content-title">"Hello! Sharing world"</p>
+                <h1>DIGITAL TRANSFORMATION<br>
+                    AFRICAN SUMMIT <br class="none">
+                </h1>
+                <div class="header-content-body">
+                    <p>The digital landscape is constantly evolving, and businesses and individuals must adapt to stay relevant.
+                       This web summit aims to explore the concept of digital transformation and how organizations can leverage technology to thrive in the future.
+                    </p>
+                </div>
+                <h3 class="summit-date">2023.09.11(WED) ~ 13(FRI)</h3>
+                <p class="summit-venue">@  Wekalet El Ghouri Arts Center Egypt</p>
+            </div>
+        </div>
+    </section>
+
+            <!-- Main Section -->
+    <section class="main-program mb-2">
+        <div class="main-program-title">
+            <p>Main Program</p>
+            <hr class="hr-line">
+        </div>
+        <div class="programs-card-container mb-3 pt-3">
+            <div class="program">
+                <img src="img/program-icon-1.png" alt="#" class="program-img">
+                <p class="program-title">Lecture</p>
+                <p class="program-description">Listen to speaker from various countries about the messages of sharing
+                    and opening.</p>
+            </div>
```

Comment:
as suggested previously you made these items obscure 🙅 , which I think is not the perfect way of making it. When you check on the Cindy design you noticed that it's clear, Kindly take a look at the images below, then make sure you solve that issue 👎

![image](https://github.com/Edgahkipkemoi/capstone/assets/60612329/5546de74-7311-4a89-b55f-3ebf1665911a)

original
![image](https://github.com/Edgahkipkemoi/capstone/assets/60612329/ee433d6f-a996-41dd-a1b6-3255b45cd1a7)